examples/tools/epp/prm-config.yml

```diff
@@ -0,0 +1,19 @@
+---
+plugin:
+  author: puppetlabs
+  id: epp
+  display: Embedded Puppet (EPP)
+  version: 0.1.0
+  upstream_project_url: https://puppet.com/docs/puppet/7/lang_template_epp.html
+
+puppet: true
+
+common:
+  can_validate: true
+  needs_write_access: false
+  use_entrypoint_script: true
```

Comment:
I think this should be a configurable string as fixing the name to `entrypoint` might skew folks to think its a docker only thing.
The value should be the name of the script without the extension. Depending on OS, the extension will be added.
e.g `use_entrypoint_script: "runme"` would look to copy `runme.sh` and `runme.ps1` from the content folder.

Reply:
updated to `use_script`, updating the design docs to match.